python/taichi/lang/struct.py

```diff
@@ -81,11 +81,19 @@ def keys(self):
            >>> a.keys
            ['center', 'radius']
         """
-        return list(self.entries.keys())
+        return list(self.__entries.keys())
 
     @property
     def _members(self):
-        return list(self.entries.values())
+        return list(self.__entries.values())
+
+    @property
+    def entries(self):
```

Comment:
Will this be overrided if there is a struct member named `entries`? BTW is this property used anywhere?

Reply:
This would be overrided in case there is a struct member named entries. In another word, this property would not take effect when there is a struct member named entries. This property has no more usages, but I'm afraid that there could be some users accessing `entries` and their program would raise errors on new Taichi. So I implemented this property to ensure compatibility.